DevCycle.SDK.Server.Cloud.Example/Program.cs

```diff
@@ -14,18 +17,60 @@ private static async Task Main(string[] args)
 
             var api = new DevCycleCloudClientBuilder().SetEnvironmentKey(SDK_ENV_VAR).SetLogger(new NullLoggerFactory()).Build();
             
+            // Below is an example for OpenFeature. This is not required to 
+            EvaluationContext ctx = EvaluationContext.Builder()
+                .Set("user_id", "test")
+                .Set("customData",
+                    new Structure(new Dictionary<string, Value> { { "customkey", new Value("customValue") } }))
+                .Set("privateCustomData",
+                    new Structure(new Dictionary<string, Value>
+                        { { "privateCustomKey", new Value("privateCustomValue") } }))
+                .Set("email", "email@email.com")
+                .Set("name", "Name Name")
+                .Set("language", "EN")
+                .Set("country", "CA")
+                .Set("appVersion", "0.0.1")
+                .Set("appBuild", 1)
+                .Set("nonSetValueBubbledCustomData", true)
+                .Set("nonSetValueBubbledCustomData2", "true")
+                .Set("nonSetValueBubbledCustomData3", 1)
+                .Set("nonSetValueBubbledCustomData4", new Value((object)null))
+                .Build();
+            
+            Api.Instance.SetProvider(api.GetOpenFeatureProvider());
+            FeatureClient oFeatureClient = Api.Instance.GetClient();
+            var allVariables = await api.AllVariables(DevCycleUser.FromEvaluationContext(ctx));
+            foreach (var readOnlyVariable in allVariables)
+            {
+                switch (readOnlyVariable.Value.Type)
+                {
+                    case "String":
+                        Console.WriteLine(readOnlyVariable.Key + " ---- "+ (await oFeatureClient.GetStringDetails(readOnlyVariable.Key, "default", ctx)).Reason);
```

Comment:
I wanted to print the reason because later on we print the value as part of the DVC Variable dump - to prevent double printing this also gives a "open-feature" esque highlight instead of just showing the value twice.